prime-router/src/main/kotlin/fhirengine/translation/hl7/schema/converter/ConverterSchemaReader.kt

```diff
@@ -2,14 +2,28 @@ package gov.cdc.prime.router.fhirengine.translation.hl7.schema.converter
 
 import gov.cdc.prime.router.fhirengine.translation.hl7.SchemaException
 import gov.cdc.prime.router.fhirengine.translation.hl7.schema.ConfigSchemaReader
+import gov.cdc.prime.router.fhirengine.translation.hl7.utils.getURI
+import java.net.URI
 
 /**
  * Read a converter schema [schemaName] from a file given the root [folder].
  * @return the validated schema
  * @throws Exception if the schema is invalid or is of the wrong type
  */
 fun converterSchemaFromFile(schemaName: String, folder: String? = null): ConverterSchema {
-    val schema = ConfigSchemaReader.fromFile(schemaName, folder, schemaClass = ConverterSchema::class.java)
+    val schemaUri = getURI(folder, schemaName)
```

Comment:
I'm not following this change either.  What case is this handling?

Reply:
getURI(folder, schemaName) is an adaptor func I added, it will adapt some existing local file system semantics based schema loading code so that they will work with schemas references in URI format.

I think those schema loading funcs with folder, schemaName parameters - should be deprecated since it is switched to resource URI semantics, but could be done later, for now I just want to make things working.

Reply:
Agreed, though there is a clean up ticket at the end of this epic that capture that work.  The scope here was to try to just get the existing schemas going through the new code path